Sources/ScipioS3Storage/Compressor.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import System
+import Compression
+import AppleArchive
+
+struct Compressor {
```

Comment:
The name of `Compressor` is a little ambigious so please rename like `AARArchiver` and add comment about AAR